PLGMAG2V2-435: Update Magento Vault (#2120)
MultiSafepay/docs: content/integration/ready-made/magento2/faq/activating-magento-vault.md

You must:  

> - Include a checkbox at checkout for customers to give permission to tokenize their payment details (Already included in Magento Vault).

Reviewer: ```suggestion
- Include a checkbox at checkout for customers to give permission to tokenize their payment details (provided as standard by Magento Vault).
```
